### DISCUSSION TITLE
Automatic sync between sandboxie-docs repo and sandboxie-plus website

### DISCUSSION THREAD

**Discussion by isaak654** [sandboxie-plus/Sandboxie#1756] [Documents and notes]
I would suggest to find a way to automatically synchronize the documentation between the [Sandboxie documentation repository](https://github.com/sandboxie-plus/sandboxie-docs) and the outdated [sandboxie-plus.com documentation](https://sandboxie-plus.com/sandboxie/helptopics), which usually contains a mix of broken links and old references.

### [List of known issues](https://github.com/sandboxie-plus/Sandboxie/discussions/1756#discussioncomment-3780799)

**Comment by isaak654** [comment]
![docs_update](https://user-images.githubusercontent.com/12372772/193446712-c7cc8db2-55c6-4b3a-ad45-5bfada258c73.png)

### Known issues:
-  integration of anchors needs to be finished -> #2564
- rendering issue with horizontal separators (see [here](https://sandboxie-plus.com/sandboxie/sandboxiecontrol/))
- no automatic sync (for manual synchronization, you will need to request it each time to @DavidXanatos)
- the manual sync doesn't take in consideration overwritten images in [sandboxie-docs/Media/](https://github.com/sandboxie-plus/sandboxie-docs/tree/main/Media)

**Reply by stdedos** [reply]
Is this relevant anymore? Also 
* https://github.com/sandboxie-plus/Sandboxie/blob/96b03335651b784e019d12823a3af1355069b6f9/.github/ISSUE_TEMPLATE/1-problem_report.yml?plain=1#L12, 
* https://github.com/sandboxie-plus/Sandboxie/blob/96b03335651b784e019d12823a3af1355069b6f9/.github/ISSUE_TEMPLATE/2-feature_request.yml?plain=1#L12

... given the https://github.com/sandboxie-plus/sandboxie-docs/pull/167 / https://sandboxie-plus.github.io/sandboxie-docs/

**Reply by stdedos** [reply]
Also, pages like e.g https://sandboxie-plus.com/privacy-mode/ - I think that https://sandboxie-plus.github.io/sandboxie-docs/PlusContent/privacy-mode/ is looking better (maybe it's my aversion to the purple edges 😅) , ... and also renders correctly (see e.g. 
![image](https://github.com/user-attachments/assets/9dcaabd9-f21d-41f1-8a03-020b5652f633)
)

Also also https://sandboxie-plus.github.io/sandboxie-docs/PlusContent/security-mode/ vs https://sandboxie-plus.com/security-mode/
